pom.xml

```diff
@@ -136,7 +136,7 @@
         <metrics-core.version>2.2.0</metrics-core.version>
         <miniball.version>1.0.3</miniball.version>
         <s4.version>0.6.0-incubating</s4.version>
-        <samza.version>0.7.0</samza.version>
+        <samza.version>0.10.1-MINE</samza.version>
```

Comment:
Yes, you're correct - my work environment details crept in as I'm getting used to the whole process.